libmscore/line.cpp

```diff
@@ -851,8 +857,10 @@ QPointF SLine::linePos(Grip grip, System** sys) const
                                                 extendToEnd = false;
                                           }
                                     }
-                              ChordRest* right = extendToEnd ? cr : toChordRest(startElement());
+#endif
```

Comment:
maybe better `#else ... #endif`?

Reply:
Will keep that in mind if I keep this code at all.